common/src/main/java/com/wynntils/core/features/properties/KeyBinds.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.features.properties;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Used to indicate that a feature has keybinds that must be registered.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface KeyBinds {}
```

Comment:
I don't like this name tbh, let's find something that better describes what's going on here. Maybe `KeyBindRegister` (although this only feels a tiny bit nicer)?

Reply:
Maybe `KeyListener` to to along with the `EventListener`? ;-)

Reply:
definitely agree that the name is subpar. i'm not sure if `KeyListener` truly captures the purpose of the annotation though, since registering the horkeys as control options is also an important aspect - but maybe that's splitting hairs